drivers/storage/portworx/components.go

```diff
@@ -0,0 +1,1092 @@
+package portworx
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"time"
+
+	corev1alpha1 "github.com/libopenstorage/operator/pkg/apis/core/v1alpha1"
+	"github.com/libopenstorage/operator/pkg/util"
+	k8sutil "github.com/libopenstorage/operator/pkg/util/k8s"
+	"github.com/portworx/sched-ops/k8s"
+	"github.com/sirupsen/logrus"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	pxServiceAccountName      = "portworx"
+	pxClusterRoleName         = "portworx"
+	pxClusterRoleBindingName  = "portworx"
+	pxRoleName                = "portworx"
```

Comment:
This is a good opportunity for consistent naming. Usually people don't name objects as *-role or *-account. We can handle this easily when we support upgrades. For new users it would be consistent easy naming.